frontend/src/lib/api.ts

```diff
@@ -882,6 +882,15 @@ export class ApiRequest {
         return this.userInterviews(teamId).addPathComponent(id)
     }
 
+    // # Issues
```

Comment:
We call groups of exceptions in error tracking "issues" so there's a potential for overlap in terms here e.g. "an issue about an issue" could be confusing. I know that's what Linear and GitHub call them so I'm guessing we don't want to stray too far from that.

Is the idea that this "issue tracker" is more akin to a project management tool? My understanding was that it would suggest many things that could be done e.g. experiment with the CTA button colour, setup feature tracking. Maybe we wouldn't want to frame all of those items as problems. Spitballing but "tasks" could be a better name?